docs/cheqd-cli/cheqd-cli-fee-abstraction.md

```diff
@@ -0,0 +1,170 @@
+# Leveraging Fee Abstraction through cheqd CLI
+
+## Overview
+
+A `cheqd-node` instance can be controlled and configured using the [cheqd Cosmos CLI](README.md).
+
+This document contains the Fee Abstraction commands for [paying for transactions](cheqd-cli-token-transactions.md) using governance-approved alternative tokens, e.g., a stablecoin such as USDC. The Fee Abstraction module routes requests for supported IBC denominations to [Osmosis DEX](https://app.osmosis.zone), and uses existing liquidity pools (e.g., the [CHEQ/USDC pool](https://app.osmosis.zone/pool/1273)) to convert the tokens on the fly. The underlying transaction on cheqd network is **always** funded in CHEQ tokens.
+
+Various commands are available for declaring fees in transactions, along with querying and configuring allowed denominations through governance proposals.
+
+## Supported IBC denominations
+
+Fees for transactions cannot be denominated in arbitrary alternative tokens, when attempting to pay for transactions with non-CHEQ tokens. Supported IBC denominations must be approved using decentralised governance.
```

Comment:
I would move what you're talking about in appendix here, under sub-headings for currently supported denoms. Also, we can't add it yet, but this section is where I'd add guidance around having balance in the associated Osmosis wallet etc.

Reply:
I think this is a good place to set expectations early, especially to less advanced audience.